GitUI/Script/FilePrompt.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Linq;
+using System.Windows.Forms;
+
+namespace GitUI.Script
+{
+    public partial class FilePrompt : Form
+    {
+        public string FileInput { get; private set; } = string.Empty;
+
+        public FilePrompt()
+        {
+            InitializeComponent();
+        }
+
+        private void btn_OK_Click(object sender, EventArgs e)
+        {
+            FileInput = txt_FilePath.Text;
```

Comment:
`DialogResult` must be set to `DialogResult.OK`
when the dialog is closed the result must be checked, if the result is not set `arguments` must not be changed

Reply:
Done.